client/src/app/components/AppDialogShell.vue

```diff
@@ -207,8 +209,13 @@ export default {
       margin-bottom: 0;
     }
   }
-  .dialog-footer-errors li {
+  .dialog-footer-errors {
     /*margin-left: 5px;*/
+    display: inline-block;
+    max-width: 62%;
```

Comment:
Interesting max-width value.  Can we explain why 62%?  Is this responsive to smaller screen sizes?

Reply:
I came to this value by doing some tinkering with different screen sizes, until I came up with this value. It works for resolutions until down to a tablet screen (I did responsive browser tests).